llvm/lib/CodeGen/TargetLoweringObjectFileImpl.cpp

```diff
@@ -75,6 +75,10 @@ static cl::opt<bool> JumpTableInFunctionSection(
     "jumptable-in-function-section", cl::Hidden, cl::init(false),
     cl::desc("Putting Jump Table in function section"));
 
+static cl::opt<bool> UniqueExplicitSections(
+    "unique-explicit-sections", cl::Hidden, cl::init(true),
```

Comment:
untested

Reply:
Removed, it was a leftover from earlier version.